todoApp/clients/javascript/vitest.config.ts

```diff
@@ -0,0 +1,33 @@
+// Licensed under the MIT License.
+
+import { defineConfig } from "vitest/config";
+import { relativeRecordingsPath } from "@azure-tools/test-recorder";
```

Comment:
Do we need this for unbranded scenario?

Reply:
@v-jiaodi  could you set generateTest as false and add samples instead of test for it ?

Reply:
updated